### ISSUE TITLE
Multi-streamer event handler not firing?

### ISSUE THREAD

**Issue by FloppyDingo** [iGoodie/TwitchSpawn#44]
Version is 0.8.1 on 1.12.2 minecraft. 

`[21:52:17] [EventThread/INFO] [net.programmer.igoodie.twitchspawn.TwitchSpawn]: Handling (for CalaMariGold) arguments {eventType=subscription, eventAccount=twitch, eventName=Twitch Subscription, streamerNickname=CalaMariGold, actorNickname=silverdragon1237, subscriptionMonths=1, subscriptionTier=1, gifted=true, chatBadges=[]}
[21:52:17] [EventThread/INFO] [net.programmer.igoodie.twitchspawn.TwitchSpawn]: Found associated ruleset for CalaMariGold. Handling with their rules
[21:52:17] [EventThread/INFO] [net.programmer.igoodie.twitchspawn.TwitchSpawn]: Queued handler for Twitch Subscription event.`

After it queues the handler it seems to do nothing. Not sure if there's a way to look deeper into the logs or not - but the streamer claims nothing happened after the subscriptions.

**Comment by FloppyDingo** [comment]
I have a ruleset for both gifted and ungifted subs

**Comment by Z0vris** [comment]
You have their credentials setup in the credentials file correctly? You can try to run a simulated sub with TwitchSpawn in game. 

Should be " /twitchspawn simulate {event:"twitch subscription"} CalaMariGold " without the quotes.

**Comment by FloppyDingo** [comment]
Yep, certain events fired fine. The rulesets are copies of each other and
the first streamer doesn't have issues. It's only the second one. And only
sporadically.

**Comment by Z0vris** [comment]
Is this being run on a server? I don't think servers can run two different rulesets. It will either run one or the other, but not both, I believe.

**Comment by Z0vris** [comment]
I just found out, you can have multiple rulesets. I was unaware of this. Are both rulesets on the server or client side? They need to all be on the server to work.

**Comment by FloppyDingo** [comment]
Everything is on the server yeah. One of the first things I did was test if
the rulesets were client side or server side to make updating easier (a lot
easier when you don't have to make people update their clients as well).

I can send you both rulesets if that would help? I tested both rulesets
using the in game test command that ran through everything and they seemed
fine.

**Comment by Z0vris** [comment]
You can send them. You tested the command while your friend was online? That would be the best way to test.

**Comment by FloppyDingo** [comment]
I'll test with them online soon. Once I get home I'll send over the files.

**Comment by FloppyDingo** [comment]
Here you go.

[TwitchSpawnRules.zip](https://github.com/iGoodie/TwitchSpawn/files/6133844/TwitchSpawnRules.zip)
